proactive-jupyter-kernel/kernel.py

```diff
@@ -254,6 +262,14 @@ def __trigger_pragma__(self, pragma_info):
             return self.__show_scheduling_portal__
         elif pragma_info['trigger'] == 'show_workflow_automation':
             return self.__show_workflow_automation__
+        elif pragma_info['trigger'] == 'get_target_env_nodesources':
```

Comment:
I propose to use switch/case, it'll be more readable

Reply:
It's a good idea @medou-boushab , but not applicable because python language does not have a switch/case control structure.

Reply:
ok